userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java

```diff
@@ -109,12 +110,15 @@
     
     private SharedPreferences mSharedPreference;
 
+    private Context mApplicationContext;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        mApplicationContext = getApplicationContext();
```

Comment:
why adding this?

Reply:
@addev-ashish do we still need mApplicationContext?